package.json

```diff
@@ -262,5 +262,8 @@
     "Jenkinsfile": "lint:jenkinsfile",
     "*package.json": "lint:package-names"
   },
-  "snyk": true
+  "snyk": true,
+  "devDependencies": {
```

Comment:
We don't have to. I'm used to including packages in dev to reduce the size. I know this is taken care of by the script for each `package.json` because they're built-to-order, but I find this to be an anti-pattern. 

I'm happy to put in dependencies, but I would like to visit this again because I find it confusing.